nestkernel/node_manager_impl.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *  node_manager_impl.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "node_manager.h"
+
+namespace nest
+{
+
+inline void
+NodeManager::prepare_node_( Node* n )
```

Comment:
The typical workload of this function through the `init_buffers()` and `calibrate()` calls will be so large that there is not really much point in inlining.